entrypoint.py

```diff
@@ -0,0 +1,123 @@
+from nose.tools import set_trace
+
+class EntryPoint(object):
+
+    """A EntryPoint is a top-level entry point into a library's Lane structure
+    that may apply additional filters to the Lane structure.
+
+    The "Books" and "Audiobooks" entry points (defined in the
+    EbooksEntryPoint and AudiobooksEntryPoint classes) are different
+    views on a library's Lane structure; each applies an additional
+    filter against Edition.medium.
+
+    Each individual EntryPoint should be represented as a subclass of
+    EntryPoint, and should be registered with the overall EntryPoint
+    class by calling EntryPoint.register.
+
+    The list of entry points shows up as a facet group in a library's
+    top-level grouped feed, and in search results. The SimplyE client
+    renders entry points as a set of tabs.
+    """
+
+    # The name of the per-library setting that controls which entry points are
+    # enabled.
+    ENABLED_SETTING = "enabled_entry points"
```

Comment:
Are you perhaps missing an underscore here?

Reply:
Yes, definitely.